packages/devtools_extensions/README.md

```diff
@@ -210,14 +210,25 @@ dart run devtools_extensions build_and_copy \
   --dest=path/to/your_pub_package/extension/devtools 
 ```
 
+Then publish your package.
+
 ### What if I don't want the `extension/devtools/build/` contents checked into source control?
 
 As a package author, the content that you check into your git repository is completely up to you.
 If you want the contents of `extension/devtools/build/` to be git ignored, then you'll just need
 to ensure that the extension web app is always built and included in `extension/devtools/build/`
-when you publish your package.
+when you publish your package. To do so, add the following to a `.pubignore` file at the root
+of your pub package:
+
+```
+!./extension/devtools/build
```

Comment:
@kenzieschmoll 
When you put a `.pubignore` at the root, then the root `.gitignore` won't be considered at all.

So that might be undesirable... or at least the user will have to duplicate patterns they want in both.

A better option could be to do `extensions/devtools/.pubignore` that way it's only for the `extensions/devtools/`.

It won't pollute root, and most likely there is no `.gitignore` in extensions/devtools anyways.